<launch>
    <node pkg="rover" type="beacon_detection.py" name="amber_spot_finder" output="screen"/>
    <node pkg="rover" type="beacon_coordinates.py" name="beacon_gps" output="screen"/>

</launch>